lib/initialize_with_config.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require_relative "meta"
+require "active_support"
+require "active_support/core_ext"
+
+module InitializeWithConfig
```

Comment:
Should this be in the calculator namespace?

Reply:
Seems sensible now it's part of a bigger thing

Reply:
It's not necessarily just for the calculator. I'd saw leave it in lib.